assignment1/kthInLinkedList.py

```diff
@@ -0,0 +1,41 @@
+class Node:
+    def __init__(self, _key = 0, _next = None):
+        self.key = _key
+        self.next = _next
+
+    def __str__(self):
+        return str(self.key)
+
+def kthInLinkedList(linkedList, k):
```

Comment:
Add a docstring for this function.

**Note**: the current implementation returns a `Node` but you probably want to return a value instead.